learning/scripting/visual_script/visual_script_basics.rst

```diff
@@ -342,19 +367,19 @@ Below the member panel, exists the list of all available node types:
 
 .. image:: img/visual_script33.png
 
-Once you're comfortable and familiar with them, the Ctrl-F (Command-F on
-Mac) is very useful for searching the list.
+
+Once you're comfortable and familiar with them, the Ctrl-F (Command-F on Mac) is very useful for searching the list.
 
 Any of them can be dragged to the scene. Remember that, unlike the nodes previously discussed (e.g. dragging a property
-from the inspector sets the context to the node being edited automatically), these are
-added without any "contextual" information, so this has to be done manually.
+from the inspector sets the context to the node being edited automatically), these are added without any "contextual" information, so this has to be done manually.
```

Comment:
- Inspector

This is too confusing for beginners (refers to previous discussion that may not have been read/retained, refers to "context" which could be confusing)